src/test/java/pages/Users.java

```diff
@@ -0,0 +1,31 @@
+package pages;
+
+public class Users {
+    protected String firstName,
+            lastName,
+            businessName,
+            phone,
+            email,
+            password;
+
+    public Users emptySignUp() {
```

Comment:
this method should placing in class UserFactory and each time return new User instance